restaurantInput.html

```diff
@@ -32,6 +32,9 @@ <h2 class="mdl-card__title-text  description-title">Enter Your Restaurants (2 mi
                             Start typing your chosen restaurants & select to enter them into the tournament!
                         </div>
                     </div>
+                    <!-- Simple MDL Progress Bar -->
+                <div id="p1" class="mdl-progress mdl-js-progress"></div>
```

Comment:
Is there supposed to be a progress bar in restaurantInput? Is it supposed to be empty & represent the start state (like the first round will have some progress when you go to tournament.html)?

Reply:
there isn't supposed to be